manifests/router.pp

```diff
@@ -25,6 +29,8 @@
   $worker_threads          = $qpid::router::params::worker_threads,
   $hello_interval          = $qpid::router::params::hello_interval,
   $hello_max_age           = $qpid::router::params::hello_max_age,
+  $service_ensure          = true,
+  $service_enable          = undef,
```

Comment:
I would have liked to see data types here. Ideally whenever you add a parameter, you use a data type.

Reply:
Given this class did not have data types already, it was either add it just for these two, add it for all (and possibly overreach the point of the PR) or stick with what was present already. I chose the latter to make the focused changed. And plan to follow up adding types.